### DISCUSSION TITLE
Commands not working, What happened?

### DISCUSSION THREAD

**Discussion by Codder-Girl** [Akshay-Vs/PassLock#35] [Q&A] [answered]
I just found that old commands are not working any more, If it's please upload the list of new commands

**Comment by akvsx** [comment, marked as answer]
### Commands are now easier to remember, and the steps required to obtain passwords are reduced.
### We have already provided a list of updated commands. 

<br>
<h2>Here is the list:</h2><br>
<code>new   </code> Create a new password<br>
<code>show  </code> Decrypt and show password<br>
<code>c    </code> Copy password to clipboard<br>
<code>r    </code> or leaving password blank will generate a random password<br>
<code>r 40 </code> Create random password of length r<br>
<code>edit</code> To edit passwords<br>
<code>help</code> For help<br>
<code>move</code> To move passwords to different folder<br>
<code>del name</code> To delete password<br>
<code>p</code> To change path<br>
<code>list</code> To list all password ids<br>
<code>exit</code> To exit console application<br>
